pkg/grub/rootfs.cfg

```diff
@@ -270,7 +270,7 @@ function set_arm64_baremetal {
       if [ "$smb_product" = "FR201" ]; then
          set_to_existing_file devicetree /boot/dtb/broadcom/bcm2711-rpi-cm4-fr201.dtb
          set_global dom0_console "console=ttyS0,115200 console=tty1 earlycon=tty1"
-         set_global dom0_platform_tweaks "eve_install_disk=mmcblk0 eve_persist_disk=sdb"
+         set_global dom0_platform_tweaks "eve_install_disk=mmcblk0 eve_persist_disk=sdb rootdelay=10"
```

Comment:
I’d personally go with `rootwait` here instead of `rootdelay=10`. It feels more reliable since it waits indefinitely for the root device to show up, which avoids timing issues entirely. `rootdelay` might make sense if there’s a fallback mechanism to handle the device not being ready in 10 seconds, but without that, I think `rootwait` is the safer and simpler choice.

Reply:
I thought about that as well, TBH I was wiling to use `rootwait` but don't you think that wait indefinitely might also be an issue? This particular device has a NVMe SSD installed through a M.2 USB adapter, that's a pretty controlled environment, 10 secs should be plenty enough time, otherwise something might be wrong in the hardware.... I do prefer to have a clear failure instead of the system got stuck so you never know if it's doing something or not....

Reply:
Ok, "fail-fast" might be the right approach here...
By the way, [since 6.6](https://kernelnewbies.org/Linux_6.6), there’s a `rootwait=` option available. I think that would be perfect! But we don't use this kernel yet...